### ISSUE TITLE
Error running after creating config file.

### ISSUE THREAD

**Issue by cevarief** [bobthecow/psysh#179]
I've got this error when running psysh after creating config file ~/.config/psysh/config.php 

~~~
PHP Fatal error:  Call to undefined method Psy\Configuration::setTabCompletionMatchers() in phar:///usr/bin/php-repl/src/Psy/Configuration.php on line 256
~~~

**Comment by bobthecow** [comment]
Oh, sorry about that. It's been fixed in develop. For now, comment out the tab completion matchers or use the dev branch? I'll try to get a release out soon.

**Comment by bobthecow** [comment]
This has been fixed in v0.4.2!
